examples/GoPlatform/README.md

```diff
@@ -32,15 +32,31 @@ file:platform/host.h
 ## Build Instructions
 
 The Roc compiler can't build a go platform by itself. [In the
-future](https://roc.zulipchat.com/#narrow/stream/304641-ideas/topic/Platform.20host.20build.20process) there will be a way for a platform to define the build process.
-For now, the compiled Roc binary and the go binary have to be manualy [linked](https://en.wikipedia.org/wiki/Linker_(computing)).
+future](https://github.com/roc-lang/roc/issues/6414) there will be a way for a
+platform to define the build process.
 
-Build steps:
+### Preprocess host
+
+First, an example roc application has to be build as libary. Then the platform
+has to be compiled to a file called `dynhost`. As last step, two files for the
+surgical linker have to be generated.
+
+This method uses [zig](https://ziglang.org/) as the c compiler used by go.
```

Comment:
Can you add the reason for this choice to the README.md?

Reply:
Do you mean in contrast of the "old" way to compile a go-platform? Or do you mean, why zig is used as the c compiler?

Reply:
Yeah, why zig is used for the c compiler.

Reply:
I was not able to do it without zig. The surgical linker throw an cryptic error message (I am currently not on my PC. I will copy an paste the error message later).

I remembered articles and talks, that when you want to combine go- an c-code, you get better results, when you use zig as c-compiler. So I tried this and it immediately worked. 

It would be great If you/we succeed to compile it without zig.

Reply:
It just works for me without zig locally on NixOS, I'll try it on CI.

Reply:
Strange. Now It also works for me. I have no idea, what I did yesterday.

Reply:
It happens :smile: